### ISSUE TITLE
validateFunc function not called when i load my  react application with webserver (Hapi js) inside Iframe 

### ISSUE THREAD

**Issue by ramusesan** [hapijs/hapi#4243]
Hapi js: 16.7.0
hapi-auth-cookie: 6.1.1

i have implemented the application react and hapi js (web server).
and i have hapi-auth-cookie plugin for session authentication on each api  request which has strategy session and cookies . 
if i load my application in separate window (browser tab) validateFunc is called properly, but if i load my application inside the Iframe its not working and rather my api returns html content of index.html rather than JSON

```
server.auth.strategy('session', 'cookie', {
  password: 'longpassword-should-be-32-characters-for-pulse',
  cookie: 'my-app-sid',
  redirectTo: '/',
  ttl: 86400000,
  isSecure: false,
  **validateFunc**: isAuth   // this is not called incase my app loaded inside iframe
});
```

**Comment by Nargonath** [comment]
@ramusesan You might want to take a look at the `sameSite` cookie setting: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite and perhaps also the `domain` setting for the `Set-Cookie` HTTP header.

Also next time please don't open multiple issues for the same problem. This is most likely a problem with your own setup/project so StackOverflow might be better suited for such question as GitHub issues are more for problem within hapi itself. Good luck. 👍

**Comment by ramusesan** [comment]
@Nargonath Nargonath

okay sorry for posting multiple times.
I can set **sameSite** none and secure.
but since cookies are setting by the  **hapi-auth-cookie: 6.1.1**  and encrypted how can i set that with cookies. 
also, i see isSameSite; false for new version @hapi/cookie.
but flag is not available in the version of hapi-auth-cookie: 6.1.1. can you suggest to me some how I can add this sameSite:none

thank you
